### PR TITLE
Adding Phoenix web event display proposal

### PR DESCRIPTION
**Pull request by EdwardMoyse**
Adding a proposal (and project) for the phoenix web event display.

### CODE REVIEW

File: _gsocprojects/2019/project_Phoenix.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Phoenix
+project: Phoenix
+layout: default
+logo:
```
Comment (JavierCVilla) [inline _gsocprojects/2019/project_Phoenix.md]: Do you have a logo to add?

File: gsoc/2019/mentors.md [hunk 2]
```diff
@@ -78,6 +78,7 @@ layout: plain
 * Joao Calado Vicente [joao.calado.vicente@cern.ch](mailto:joao.calado.vicente@cern.ch) CERN
 * Tobias Wegner [tobias.wegner@cern.ch](mailto:tobias.wegner@cern.ch) CERN
 * Sandro Wenzel [sandro.wenzel@cern.ch](mailto:sandro.wenzel@cern.ch) CERN
+<<<<<<< HEAD
```
Comment (JavierCVilla) [inline gsoc/2019/mentors.md]: Can you please remove this line?

File: gsoc/2019/mentors.md [hunk 3]
```diff
@@ -90,3 +91,6 @@ layout: plain
 * Axel Naumann [axel.naumann@cern.ch](mailto:axel.naumann@cern.ch) CERN
 * Frank Wuerthwein [fkw@ucsd.edu](mailto:fkw@ucsd.edu) UCSD
 * Omar Zapata [Omar.Zapata@cern.ch](mailto:Omar.Zapata@cern.ch) OProject
+* Edward Moyse [edward.moyse@cern.ch](mailto:edward.moyse@cern.ch) University of Massachusetts, Amherst)
+* Riccardo Maria Bianchi  [riccardo.maria.bianchi@cern.ch](mailto:riccardo.maria.bianchi@cern.ch) University of Pittsburgh)
+* Andreas Salzburger  [andreas.salzburger@cern.ch](mailto:andreas.salzburger@cern.ch) CERN)
```
Comment (JavierCVilla) [inline gsoc/2019/mentors.md]: I think Andreas is already present on this list, you can remove it.

**Comment by aflp91** [comment]
Everything seems OK but a missing HSF project page

File: _gsocprojects/2019/project_Phoenix.md (hunk 1 above)
Comment (EdwardMoyse) [reply _gsocprojects/2019/project_Phoenix.md]: Not yet, no.

File: gsoc/2019/mentors.md (hunk 2 above)
Comment (EdwardMoyse) [reply gsoc/2019/mentors.md]: Argh. Sorry!

File: gsoc/2019/mentors.md (hunk 3 above)
Comment (EdwardMoyse) [reply gsoc/2019/mentors.md]: Done. BTW he is repeated earlier, as is James Catmore.

Comment (EdwardMoyse) [reply gsoc/2019/mentors.md]: FYI:

imacumass1:hsf.github.io emoyse$ sort gsoc/2019/mentors.md  | uniq -d

* Alaettin Serhan Mete [amete@cern.ch](mailto:amete@cern.ch) UCIrvine
* Andreas Salzburger [andreas.salzburger@cern.ch](mailto:andreas.salzburger@cern.ch) CERN
* Davide Costanzo [davide.costanzo@gmail.com](mailto:davide.costanzo@gmail.com) The University of Sheffield
* James Catmore [james.catmore@cern.ch](mailto:james.catmore@cern.ch) University of Oslo
* Paul Gessinger [paul.gessinger@cern.ch](mailto:paul.gessinger@cern.ch) CERN

File: _gsocproposals/2019/proposal_Phoenix.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+[Phoenix](https://github.com/HSF/phoenix) is an attempt at a framework that could be used by any typical (e.g. colliding beam) High Energy Physics experiment. It is based on work done for the TrackML [Kaggle](https://www.kaggle.com/c/trackml-particle-identification)/[Codalab](https://competitions.codalab.org/competitions/20112) challenges (and internal use by ATLAS), so the underlying principal is well validated but it now needs to be extended with a more sophisticated GUI, better visualisation techniques, unit tests etc. The common event data format also needs to be tested/extended with data from experiments other than ATLAS and TrackML.
```
[the remark this answers is not in the document]
Comment (EdwardMoyse) [reply _gsocproposals/2019/proposal_Phoenix.md]: Yes, thanks!

**Pull request merged**